changelog/db.changelog-1.10.xml

```diff
@@ -30,4 +30,4 @@
 <changeSet id="10" author="Leigh Godson">
     <modifyDataType tableName="delinkedCalculation" columnName="calculationId" newDataType="INTEGER"/>
 </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+</databaseChangeLog>
```

Comment:
still a bit concerned about this not lining up properly although I've got no idea what the problem is. are the latest changes up all the way to Prod? if its only up as high as Test we might be covered that we could wipe the change log history, albeit that's not a great solution

Reply:
never mind, teach me to check on 'commits since last review' - looks like its fine